tabbycat/templates/base.html

```diff
@@ -22,6 +22,20 @@
   <meta property="og:description" content="{{ meta_description }}" />
   <meta property="og:image" content="{% static 'logo-social.png' %}" />
 
+  {# Set noindex property if configured. #}
+  {% if tournament %}
+    {% if pref.search_engine_indexing == "all" %}
+    {% elif pref.search_engine_indexing == "homepage-only" %}
+        {% tournamenturl 'tournament-public-index' as tournament_homepage_url %}
+        {% if request.path == tournament_homepage_url %}
+        {% else %}
+            <meta name="robots" content="noindex">
+        {% endif %}
+    {% else %}
+        <meta name="robots" content="noindex">
+    {% endif %}
```

Comment:
I would try simplifying the logic and scoping it with a block

```suggestion
    {% if pref.search_engine_indexing != "all" %}
      {% block noindex %}<meta name="robots" content="noindex">{% endblock %}
    {% endif %}
```

Then for the tournament index page, you can add something like:

```html
{% if pref.search_engine_indexing == "homepage-only" %}{% block noindex %}{% endblock %}{% endif %}
```

to override the default here.